docarray/proto/__init__.py

```diff
@@ -1,11 +1,23 @@
-from docarray.proto.pb2.docarray_pb2 import (
-    DocumentArrayProto,
-    DocumentArrayStackedProto,
-    DocumentProto,
-    NdArrayProto,
-    NodeProto,
-    UnionArrayProto,
-)
+from google.protobuf import __version__ as __pb__version__
+
+if __pb__version__.startswith('4'):
+    from docarray.proto.pb.docarray_pb2 import (
+        DocumentArrayProto,
+        DocumentArrayStackedProto,
+        DocumentProto,
+        NdArrayProto,
+        NodeProto,
+        UnionArrayProto,
+    )
+else:
+    from docarray.proto.pb2.docarray_pb2 import (
+        DocumentArrayProto,
+        DocumentArrayStackedProto,
+        DocumentProto,
+        NdArrayProto,
+        NodeProto,
+        UnionArrayProto,
+    )
```

Comment:
aren't these two exactly the same? or am I blind? 😅

Reply:
Oh i see the difference now, nevermind

Reply:
this PR should be named "spot the difference"

Reply:
nerd version of "where is waldo?"